docs/API Reference.md

```diff
@@ -46,7 +46,7 @@ This API is a simple retrieval for a single user account
 
 #### Route
 
-/user
```

Comment:
is this how it's working now? i thought the id is passed in the body and the apikey/secret key are hashed and put into the user auth header